### DISCUSSION TITLE
Ignore attribute leads to None no matter whether it's been set or not in mock

### DISCUSSION THREAD

**Discussion by TommyLike** [SeaQL/sea-orm#1844] [Q&A]
## Description
When model attributes is defined within ignore, mock append_query_results will unable to generate these attributes, see below:
```
pub struct Model {
    #[sea_orm(primary_key)]
    pub id: i32,
    pub name: String,
    #[sea_orm(ignore)]
    pub user_email: Option<String>,
}

//create Model with append_query_results
let db = MockDatabase::new(DatabaseBackend::MySql)
            .append_query_results([
                vec![dto::Model {
                    id: 1,
                    name: "Test Key".to_string(),
                    user_email: Some("fake_email".to_string()),
                }],
            ]).into_connection();
```
the issue is we will always get `user_email` equal to **None** in the testcases.
My question is how does ignore attribute work and the result above is a designed behaviour?

## Steps to Reproduce

1. 
2. 
3.

**Comment by tyt2y3** [comment]
I think this is the intended behaviour. Because the `ignore` attribute really means it will be ignored everywhere. The value will be filled with `Default::default`.

I have an idea to add an attribute that only ignore it on `select`, but actually allow its value to be populated by query results.

**Reply by TommyLike** [reply]
Is it supported now and how?
